internal/driver/controllerserver_helper.go

```diff
@@ -103,6 +104,10 @@ type VolumeParams struct {
 func (cs *ControllerServer) canAttach(ctx context.Context, instance *linodego.Instance) (canAttach bool, err error) {
 	log := logger.GetLogger(ctx)
 	log.V(4).Info("Checking if volume can be attached", "instance_id", instance.ID)
+	if !observability.SkipObservability {
```

Comment:
nit: could be move this skip observability check inside the StartFunctionSpan()

Reply:
I tried that, but that causes a nil pointer issue. Due to which I later discussed it with Evan and moved it here.